src/plugins/intel_gna/src/gna_plugin.cpp

```diff
@@ -202,102 +205,176 @@ void GNAPlugin::copyInputData(T *dst,
     }
 }
 
-void GNAPlugin::ExportScores(void *ptr_dst,
-                  const void *ptr_src,
-                  intel_dnn_orientation_t orientation,
-                  uint32_t num_frames,
-                  uint32_t num_group,
-                  uint32_t num_vector_elements,
-                  uint32_t num_active_elements,
-                  uint32_t num_vector_stride,
-                  Precision precision_in,
-                  Precision precision_out) {
+void GNAPlugin::ExportScores(void* ptr_dst,
+                             const void* ptr_src,
+                             intel_dnn_orientation_t orientation,
+                             uint32_t num_frames,
+                             uint32_t num_group,
+                             uint32_t num_vector_elements,
+                             uint32_t num_active_elements,
+                             uint32_t num_vector_stride,
+                             Precision precision_in,
```

Comment:
please consider const& instead, here and below (+ including signature of ImportFrames)